standalone esxi host crash

Hi Brandon - fantastic work on this exporter!

I am trying to connect to a standalone esxi host (rather than vcenter), but as soon as I request `/metrics` the container crashes with the following:

```
+ exec app
INFO[0000] Starting Server: :9155                        file=vmware_exporter.go line=192
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x40 pc=0x54d8b8]

goroutine 32 [running]:
panic(0xf7ce00, 0xc82000a0b0)
        /usr/local/go/src/runtime/panic.go:481 +0x3e6
github.com/prometheus/client_golang/prometheus.NewConstMetric(0x0, 0x2, 0x0, 0xc8201e6690, 0x3, 0x3, 0x0, 0x0, 0x0, 0x0)
        /go/src/github.com/prometheus/client_golang/prometheus/value.go:156 +0x48
github.com/prometheus/client_golang/prometheus.MustNewConstMetric(0x0, 0x2, 0x0, 0xc8201e6690, 0x3, 0x3, 0x0, 0x0)
        /go/src/github.com/prometheus/client_golang/prometheus/value.go:170 +0x7a
main.(*Exporter).Collect(0xc82000c4e0, 0xc820013b60)
        /go/src/app/vmware_exporter.go:164 +0x910
github.com/prometheus/client_golang/prometheus.(*Registry).Gather.func2(0xc82038b1c0, 0xc820013b60, 0x7f2311ae11f0, 0xc82000c4e0)
        /go/src/github.com/prometheus/client_golang/prometheus/registry.go:433 +0x58
created by github.com/prometheus/client_golang/prometheus.(*Registry).Gather
        /go/src/github.com/prometheus/client_golang/prometheus/registry.go:434 +0x360
```

--- a/vmware_exporter.go
+++ b/vmware_exporter.go
@@ -160,8 +160,10 @@ func (e *Exporter) Collect(ch chan<- prometheus.Metric) {
 			for _, baseSeries := range metric.Value {
 				series := baseSeries.(*types.PerfMetricIntSeries)
 				desc := countersInfoMap[int(series.Id.CounterId)]
-				ch <- prometheus.MustNewConstMetric(desc,
-					prometheus.GaugeValue, float64(series.Value[0]), hostName, series.Id.Instance, metric.Entity.Type)
+				if desc != nil {
+					ch <- prometheus.MustNewConstMetric(desc,
+						prometheus.GaugeValue, float64(series.Value[0]), hostName, series.Id.Instance, metric.Entity.Type)
+				}
 			}
 		}
 	}

